pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -173,16 +174,25 @@ func (c *Client) GetOperatorVersion(name, namespace string) (*v1beta1.OperatorVe
 }
 
 // UpdateInstance updates operatorversion on instance
-func (c *Client) UpdateInstance(instanceName, namespace string, operatorVersionName *string, parameters map[string]string) error {
+func (c *Client) UpdateInstance(instanceName, namespace string, operatorVersion *string, parameters map[string]string, triggeredPlan *string) error {
```

Comment:
I wonder why are we keeping this method as one 🤔 Why is it better than having `TriggerPlan(instanceName, namespace string, triggeredPlan *string)`?

Reply:
I guess we could have another method 🤷‍♂  Why is it better?

Reply:
Oh, one reason to keep in the same method: we already planing on triggering plans **and** modifying parameters so you'll need both 😉